src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -970,6 +1010,95 @@ public with sharing class CRLP_RollupUI_SVC {
 
     }
 
+    /******************************************************************************************************************
+    * @description builds a map of rollups that have changed
+    * @param rollups a set of rollups
+    * @return Map<String, Object> map of rollup of set of strings
+    /*****************************************************************************************************************/
+    private static Map<String, Object> buildModifiedRollupMap(Set<String> rollups) {
+        return new Map<String, Object> {
+            CRLP_ApiService.PARAM_ROLLUP_TYPES => rollups
+        };
+    } 
+
+    /******************************************************************************************************************
+    * @description builds a set of rollup types that have changed for a filtergroup
+    * @param newFilterGroup FilterGroup object that we want to check for modifications
+    * @return Set<String> of rollup types
+    /*****************************************************************************************************************/
+    private static Set<String> buildModifiedRollupSet(CRLP_RollupCMT.FilterGroup newFilterGroup) {
+        
+        List<Filter_Group__mdt> allFilterGroupsMdt = CMT_FilterRule_SEL.getFilterGroups();
+        Map<Id, List<Filter_Rule__mdt>> rulesByGroup = CMT_FilterRule_SEL.getFilterRulesByGroup();
+        Map<Id, List<Rollup__mdt>> rollupsByFilterGroup = CRLP_Rollup_SEL.getRollupsByFilterGroup();
+
+        Set<String> modifiedRollupNames = new Set<String>();
+
+        for (Filter_Group__mdt existingFilterGroupMdt : allFilterGroupsMdt) {
+            CRLP_RollupCMT.FilterGroup existingFilterGroup = new CRLP_RollupCMT.FilterGroup(existingFilterGroupMdt);
+            
+            if (newFilterGroup.recordId == existingFilterGroup.recordId) {
+                Boolean isFilterGroupStale = newFilterGroup.isDifferent(
```

Comment:
Same note as above - `Stale` is a DSO-NPSP bridge concept. From NPSP, it's more about identifying when the Filter Group/Rules has changed and taking some action.